src/packages/step/step.harmony.css

```
@@ -0,0 +1,201 @@
+/* #ifdef harmony */
+/* #endif */
+/* #ifndef harmony */
+/* #endif */
+.nut-step {
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex: 1;
```

Biome (1.9.4)

[error] Unexpected shorthand property flex after flex-grow



(lint/suspicious/noShorthandPropertyOverrides)